packages/react-liveness/src/components/FaceLivenessDetector/service/utils/getFaceMatchState.ts

```diff
@@ -0,0 +1,117 @@
+import {
+  LivenessOvalDetails,
+  Face,
+  FaceMatchState,
+  BoundingBox,
+} from '../types';
+import { SessionInformation } from '@aws-sdk/client-rekognitionstreaming';
+import {
+  generateBboxFromLandmarks,
+  getIntersectionOverUnion,
+  getOvalBoundingBox,
```

Comment:
Are these utils are only used by `getFaceMatchStateInLivenessOval`? If so let's move them in to this file

Reply:
They're also used in the state machine https://github.com/aws-amplify/amplify-ui/blob/c1d70157ceda539e5c55dcf59f446ccc1e022686/packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts#L10